controllers/csiaddons/reclaimspacecronjob_controller_test.go

```diff
@@ -203,7 +204,7 @@ func TestGetNextSchedule(t *testing.T) {
 				t.Errorf("getNextSchedule() error = %v, wantErr %v", err, tt.wantErr)
 				return
 			}
-			if !reflect.DeepEqual(gotLastMissed, tt.lastMissed) {
+			if !reflect.DeepEqual(gotLastMissed, tt.lastMissed) && !reflect.DeepEqual(gotLastMissed, time.Time{}) {
```

Comment:
Instead of using `reflect`, you could use https://pkg.go.dev/time#Time.Equal maybe?

Reply:
Done.